src/test/scala/nl.knaw.dans.easy.multideposit/actions/AddBagToDepositSpec.scala

```diff
@@ -21,11 +21,11 @@ import java.security.MessageDigest
 import gov.loc.repository.bagit.hash.StandardSupportedAlgorithms
 import nl.knaw.dans.easy.multideposit.model.{ Deposit, DepositId }
 import nl.knaw.dans.easy.multideposit.{ Settings, UnitSpec, _ }
-import org.scalatest.BeforeAndAfter
+import org.scalatest.BeforeAndAfterEach
 
-import scala.util.Success
+import scala.util.{ Failure, Success }
 
-class AddBagToDepositSpec extends UnitSpec with BeforeAndAfter {
+class AddBagToDepositSpec extends UnitSpec with BeforeAndAfterEach {
```

Comment:
`BeforeAndAfterEach` gives better extensibility and is a better code-style overall.